UI.py

```diff
@@ -0,0 +1,109 @@
+from getpass import getpass
+# importing 'mysql.connector' as mysql 
+import mysql.connector as mysql
+
+class DbConnectionError(Exception):
+    pass
+
+#DB Connection
```

Comment:
All db connection things to be moved to db_utils

Reply:
I'll do that over the weekend :)